tests/queries/test_slippage_investigation.py

```diff
@@ -66,6 +66,66 @@ def test_no_outrageous_slippage(self):
         for obj in top_five_negative + top_five_positive:
             assert abs(int(obj["eth_slippage_wei"])) < 1 * 10**18
 
+    def test_simple_positive_slippage(self):
+        """
+        Transaction 0x2509839cda8a4fec0c3b0ae4e90537aef5afaaeeee88427161d1b08367aeb0d7
+        contains a single UniV3 interaction and resulted in 82 USDC positive slippage.
+        """
+        period = AccountingPeriod("2022-12-16", 1)
+        query = QUERIES["PERIOD_SLIPPAGE"].with_params(
+            period.as_query_params()
+            + [
+                # Default values (on the query definition) do not need to be provided!
+                QueryParameter.text_type(
+                    "TxHash",
+                    "0x2509839cda8a4fec0c3b0ae4e90537aef5afaaeeee88427161d1b08367aeb0d7",
+                ),
+                QueryParameter.text_type("CTE_NAME", "results_per_tx"),
+            ],
+            dune_version=DuneVersion.V2,
+        )
+        results = exec_or_get(self.dune, query, result_id="01GP3A0QV1BNWF55Z5N362RK8M")
+        tx_slippage = results.get_rows()[0]
+        print(tx_slippage)
```

Comment:
SHould we remove this?

Reply:
I think its kinda nice. When you run the tests, you might find the same.

Reply:
but sure, why not.
```suggestion
```